packages/browser/src/browser/index.ts

```diff
@@ -336,6 +337,11 @@ async function loadAnalytics(
     }
   }
 
+  if (options.initialPageview) {
```

Comment:
Wondering if this should be moved even higher - above the `loadLegacySettings` maybe? Basically before any async work to make it harder for the page context to be incorrect due to navigation.

Reply:
Done!